.github/workflows/tld-update.yml

```
@@ -26,7 +26,7 @@ jobs:
           go-version: 'stable'
 
       - name: Run unit tests
-        run: go test tools/*.go
+        run: go test -C ./tools .
```

While you're here would you mind adding a `-v` flag? I think the verbose output is more meaningful for a CI task.

```suggestion
        run: go test -v -C ./tools .
```